<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
    <line>
      <![CDATA[Facebook – log in or sign up]]>
    </line>
    <line>
      <![CDATA[https://www.facebook.com/]]>
    </line>
  </reporter-output>
  <suite name="Default suite" duration-ms="37759" started-at="2018-10-01T07:33:55Z" finished-at="2018-10-01T07:34:33Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="37759" started-at="2018-10-01T07:33:55Z" finished-at="2018-10-01T07:34:33Z">
      <class name="Testng.P">
        <test-method status="PASS" signature="openapp()[pri:0, instance:Testng.P@646d64ab]" name="openapp" is-config="true" duration-ms="37092" started-at="2018-10-01T07:33:55Z" finished-at="2018-10-01T07:34:32Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openapp -->
        <test-method status="FAIL" signature="login()[pri:0, instance:Testng.P@646d64ab]" name="login" duration-ms="391" started-at="2018-10-01T07:34:32Z" finished-at="2018-10-01T07:34:33Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The following asserts failed:
expected [gdshjgjkkjgkjhgfhjgfhhkjhhk] but found [Facebook – log in or sign up]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [gdshjgjkkjgkjhgfhjgfhhkjhhk] but found [Facebook – log in or sign up]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
at Testng.P.login(P.java:36)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[Facebook – log in or sign up]]>
            </line>
            <line>
              <![CDATA[https://www.facebook.com/]]>
            </line>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method status="PASS" signature="closeapp()[pri:0, instance:Testng.P@646d64ab]" name="closeapp" is-config="true" duration-ms="256" started-at="2018-10-01T07:34:33Z" finished-at="2018-10-01T07:34:33Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeapp -->
      </class> <!-- Testng.P -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
